code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -562,7 +562,7 @@
 		playsound(loc, hugsound, 25, 0)
 	if(!sterile && !issynth(user) && !isIPC(user))
 		user.disable_lights(sparks = TRUE, silent = TRUE)
-		var/stamina_dmg = user.maxHealth * 2 + user.max_stamina_buffer
+		var/stamina_dmg = user.maxHealth * 1 + user.max_stamina_buffer
```

Comment:
You can remove the `* 1`.

Reply:
ngl, what this 1 or 2 means exactly, i tested it and i just found out it halved the stam damage with is what i was looking for, but i didint understood what it really is